docs/tutorials/ad_monetization.md

```diff
@@ -147,7 +147,7 @@ If you are using Shaka's UI library, we will automatically hook up our ad UI.
 #### Listening To Ad Events
 We unify Server Side and Client Side ad events into our own Shaka ad events and
 objects. which your application can listen to and interact with.
-Check out the [full list of ad events][] for details.
+Check out the {@link shaka.ads.AdManager#event:AdBreakReadyEvent|full list of ad events} for details.
```

Comment:
Please wrap each line with 80 characters for code width. (I use the ruler in my text editor to check that.) Thank you!